cs/mailing.texy

```diff
@@ -163,10 +163,22 @@ V šabloně pak odkaz vytvoříme pomocí makra `{link}` tak, jak jsme zvyklí.
 <a href="{link Presenter:action}">Odkaz</a>
 \--
 
-Odesílání přes SMTP
-===================
+Typy mailerů
+============
 
-Výchozí mailer((třída zajišťující odesílání e-mailů)) používá PHP funkci [php:mail]. Pokud bychom potřebovali poštu odesílat skrze SMTP server, použijte `SmtpMailer`
+- SendmailMailer
+- SmtpMailer
+- FallbackMailer
+
+Výchozí mailer((třída zajišťující odesílání e-mailů)) je SendmailMailer, který používá PHP funkci [php:mail].
+
+Můžeme si také vytvořit vlastní mailer, jedná se o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+
+
+SMTP Mailer
+-----------
+
+Pokud bychom potřebovali poštu odesílat skrze SMTP server, použijte `SmtpMailer`.
```

Comment:
*K odesílání pošty přes SMTP servery slouží `SmtpMailer`* mi zní asi líp, ale nechám to na tobě.